src/ClientData/include/ClientData/TrackData.h

```diff
@@ -13,7 +13,7 @@
 
 namespace orbit_client_data {
 
-class TrackData {
+class TrackData final {
```

Comment:
nit. could be useful to add a comment as to why this class is final. I don't see virtual methods.

Reply:
so that nobody accidentally inherits it :)